Files/Views/SettingsPages/About.xaml

```diff
@@ -67,8 +70,8 @@
                                 <TextBlock
                                     x:Uid="SettingsAboutSubmitFeedbackDescription"
                                     Foreground="Gray"
-                                    Text="Submit an issue report to provide the developers with information"
-                                    TextWrapping="Wrap" />
+                                    Text="Send the developers an issue report with more information"
```

Comment:
Unless this is changed in the en-US resource file, the text will show the changes made here.